releasenotes/notes/automatic-user-login-events-ded104398ff85a1d.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: >
+  - Datadog has added support for automatically creating login success or failure events when a configured Django authentication backend is used. 
+    configured Django authentication backend are used. This will automatically fill the following tags in these cases:
```

Comment:
I'm not sure this makes sense to me.

Reply:
Just noticed the repeated sentence, fixed.